dags/htan-nf_dcqc_dag.py

```diff
@@ -48,18 +60,20 @@ def launch_tower_workflow(workspace_id: str, s3_uri: str):
             workspace_id (str): Workspace ID for tower run
             s3_uri (str): Path to S3 location of input file
         """
-        tower_utils = create_and_open_tower_workspace(platform="sage-dev", workspace_id=workspace_id)
+        tower_utils = create_and_open_tower_workspace(
+            platform="sage-dev", workspace_id=workspace_id
```

Comment:
We should tackle the Nextflow Tower hook sooner rather than later so you can get rid of hard-coded values like `sage-dev`, which hurt the portability of the DAG.

Reply:
I agree, especially considering most of our DAGs launch tower workflows